project/Dependencies.scala

```diff
@@ -0,0 +1,12 @@
+import sbt.Keys.libraryDependencies
+import sbt._
+
+object Dependencies {
+
+  val Scala212      = "2.12.10"
+  val Scala213      = "2.13.2"
+  val ScalaVersions = Seq(Scala212, Scala213)
+
+  val PlayVersion = sys.props.getOrElse("play.version", sys.env.getOrElse("PLAY_VERSION", "2.8.0"))
+
+}
```

Comment:
I'm guessing this means `master` was not releasing `sbt-0.13` versions of the `sbt-twirl` plugin, correct?

Reply:
yes

Reply:
Just saw https://github.com/playframework/twirl/pull/335#discussion_r429953809